src/groupeddataframe/splitapplycombine.jl

```diff
@@ -65,7 +62,9 @@ function _combine_prepare(gd::GroupedDataFrame,
 
     idx, valscat = _combine(gd, cs_norm, optional_transform, copycols, keeprows, renamecols)
 
-    !keepkeys && ungroup && return valscat
+    if !keepkeys && ungroup
```

Comment:
I think the previous code was idiomatic Julia style. Why do you want to change it?